udata/core/dataset/api.py

```diff
@@ -292,6 +296,45 @@ def post(self):
         return dataset, 201
 
 
+@ns.route("/recent.atom", endpoint="recent_datasets_atom_feed")
+class DatasetsAtomFeedAPI(API):
+    @api.doc("recent_datasets_atom_feed")
+    def get(self):
+        feed = Atom1Feed(
+            _("Derniers jeux de données"),
```

Comment:
I think we should use English in udata keys for consistency

Reply:
Done in 4d73be89a